server/models/Appointment.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const appointmentSchema = new mongoose.Schema({
+  meeting_id: {
+    type: String,
+    required: true,
+    unique: true
+  },
```

Comment:
We should be referencing to the meeting collection, please use `mongoose.Types.ObjectId` for the property type and add a `ref: 'meeting'`

Reply:
Also, camelCase and not snake_case here please 😄

Reply:
Just implemented your suggestions!